packages/jaeger-ui/src/components/common/TraceId.tsx

```diff
@@ -26,7 +27,11 @@ export function TraceId({ traceId, className = '' }: Props) {
   const traceIdDisplay = traceId ? traceId.slice(0, traceIdDisplayLength) : '';
   const lengthClass = traceIdDisplayLength === 7 ? 'TraceIDLength--short' : 'TraceIDLength--full';
 
-  return <small className={`TraceIDLength ${lengthClass} u-tx-muted  ${className} `}>{traceIdDisplay}</small>;
+  return (
+    <ClickToCopy text={traceId} className={`TraceIDLength ${lengthClass} u-tx-muted  ${className}`}>
```

Comment:
why does ClickToCopy need to use the same classes as the underlying widget? Why not nest whatever was there previously?
```
  <ClickToCopy textToCopy={traceId>
    <small className={`TraceIDLength ${lengthClass} u-tx-muted  ${className} `}>{traceIdDisplay}</small>
  </ClickToCopy>
```